.gitignore

```diff
@@ -0,0 +1,3 @@
+target
```

Comment:
Maybe use .gitignore used in other projects? E.g.:
https://github.com/data-integrations/google-cloud/blob/release/0.13/.gitignore